providers/okta/html.go

```diff
@@ -0,0 +1,83 @@
+package okta
```

Comment:
This is a duplication of code in Duo. "A better copying is better than a little dependency"

Reply:
> A better copying is better than a little dependency

Not sure about that :)

Reply:
yeah...it's also nasty code so having to maintain it in two places feels a bit bad. Why not a `utils` package or smth so that both providers can use it?